chevah_build

```diff
@@ -202,6 +202,9 @@ case $OS in
         export CXX="clang++"
         export CFLAGS="$CFLAGS -mmacosx-version-min=10.8"
         export MACOSX_DEPLOYMENT_TARGET=10.8
+        # The location of the Homebrew OpenSSL libs and include directories.
+        export LDFLAGS="$LDFLAGS -L/usr/local/opt/openssl/lib"
```

Comment:
It compiles fine on 10.8, I tested on the builder as I mentioned.

Apple removed the headers from the OS at some release, so I used Homebrew which will be the easiest way for customers.